inspirehep/modules/theme/static/scss/base/header.scss

```diff
@@ -48,15 +56,56 @@ body {
     }
 }
 
+.custom-dropdown-header{
+    color: #428bca;
```

Comment:
I see there's a variable with this color, and a few other uses: https://github.com/inspirehep/inspire-next/search?utf8=%E2%9C%93&q=%23428bca